modules/ROOT/pages/troubleshoot-sso-errors.adoc

```diff
@@ -53,20 +53,69 @@ To troubleshoot this issue, add the external identity provider group mapping in
 . From *Provider Name*, select the name of the corresponding Salesforce identity provider.
 . From *Type*, select *Member* and click *Add*.
 
-== Auth Provider Default Scopes Field Is Not Configure Properly
-
+* The *Auth provider Default Scopes* field is not configured properly.
++
 To troubleshoot this issue, ensure the *Default Scopes* field has the *profile openid email groups* value.
 
 . In *Salesforce*, go to *Setup*.
 . In the *Quick Find* box, enter `Auth`, and then select *Auth Providers*.
 . Click *Edit*.
-. In the *Auth Provider Detail* section, enter `*profile openid email groups*`.
+. In the *Auth Provider Detail* section, enter `profile openid email groups`.
 . Click *Save*.
 
-== Cannot Log In
+== Can't Log In
+When logging in to the portal with an identity provider, you see a *Problem Logging In* error that the third-party identifier can’t be found. An incorrect portal URL can cause issues logging in.
+
+To troubleshoot this issue, ensure that the URL is correct in the configuration. For Okta, the `/userinfo` part of the URL path must be lowercase.
 
-When trying to log in to the portal with an identity provider, you see a *Problem Logging In* error that the third-party identifier cannot be be found. An incorrect portal URL can cause issues logging in.
+== Single Sign-on Error
+After logging on, you see a *Single Sign-On Error* in the portal. Issues can occur when the SSO or the Salesforce Auth. Provider configuration is incorrect or missing necessary information. 
```

Comment:
```suggestion
After logging in, *Single Sign-On Error* shows in the portal. Issues can occur if the SSO or the Salesforce Auth. Provider configuration is incorrect or missing necessary information. 
```

Reply:
Is it really called "Salesforce Auth. Provider"?

Reply:
yep